runelite-client/src/main/java/net/runelite/client/plugins/config/PluginHubPanel.java

```diff
@@ -140,6 +137,18 @@
 			manifest = newManifest == null ? loaded : newManifest;
 			this.installed = installed;
 
+			if (manifest != null)
+			{
+				Collections.addAll(keywords, SPACES.split(manifest.getDisplayName().toLowerCase()));
+				Collections.addAll(keywords, SPACES.split(manifest.getDescription().toLowerCase()));
```

Comment:
this can throw npe, and I'm not sure its even a good idea

Reply:
Updated (and marked description/tags as nullable). And at least for now imo it is fine (as long as external plugin descriptions make sense, and atm most of them do).